persist/composite/manager.go

```diff
@@ -1,183 +0,0 @@
-// Copyright (c) 2016 Uber Technologies, Inc.
```

Comment:
No it's not, and probably won't ever be. It's there because at one point we thought we were gonna dual write to lcoal fs and C*, but that turned out unnecessary, so now it just becomes a burden to maintain every time we need to change the `Manger` interface